src/app/kuma/components/ApplicationShell.vue

```diff
@@ -0,0 +1,280 @@
+      >
+        <div class="app-notifications">
+          <slot name="notifications" />
+        </div>
+        <slot
+          name="notifications"
+        >
+          <KAlert
+            v-if="!can('use state')"
+            class="mb-4"
+            appearance="warning"
+          >
+            <template #alertMessage>
+              <ul>
+                <!-- eslint-disable vue/no-v-html  -->
+                <li
+                  data-testid="warning-GLOBAL_STORE_TYPE_MEMORY"
+                  v-html="t('common.warnings.GLOBAL_STORE_TYPE_MEMORY')"
+                />
+                <!-- eslint-enable -->
+              </ul>
+            </template>
+          </KAlert>
+          <OnboardingNotification />
+        </slot>
+        <slot name="default" />
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts" setup>
+import { KUI_ICON_SIZE_30 } from '@kong/design-tokens'
+import { CogIcon, HelpIcon } from '@kong/icons'
+import GithubButton from 'vue-github-button'
+
+import { useEnv, useI18n, useCan } from '@/app/application'
+import UpgradeCheck from '@/app/common/UpgradeCheck.vue'
+import OnboardingNotification from '@/app/onboarding/components/OnboardingNotification.vue'
+
+const env = useEnv()
+const can = useCan()
+const { t } = useI18n()
+
+</script>
+<style lang="scss">
+html.is-fullscreen {
+  .application-shell > header {
+    display: none;
+  }
+  .application-shell > .app-content-container > nav {
+    display: none;
+  }
+  .application-shell > .app-content-container > .app-notifications {
+    display: none;
+  }
+}
+html.no-navigation {
+  .application-shell > .app-content-container > nav {
+    display: none;
+  }
+}
+</style>
+
+<style lang="scss" scoped>
+.app-content-container {
+  padding-top: var(--AppHeaderHeight, initial);
+  display: var(--AppDisplay);
+  grid-template-columns: var(--AppSidebarWidth) 1fr;
+}
+
+.app-main-content {
+  padding: var(--AppContentPadding);
+}
+
+header {
+  position: fixed;
+  z-index: 11;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: var(--AppHeaderHeight);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: $kui-space-80;
+  padding-right: $kui-space-80;
+  padding-left: $kui-space-80;
+  border-bottom: $kui-border-width-10 solid $kui-color-border;
+  background-color: $kui-color-background;
+}
+
+.horizontal-list {
+  display: flex;
+  align-items: center;
+  gap: $kui-space-80;
+}
+// This wrapping element is necessary. It ensures that the sidebar can participate in a grid or flex container.
+nav {
+  position: static;
+}
+
+nav ul {
+  list-style-type: none;
+  padding: 0;
+  width: var(--AppSidebarWidth);
+  position: fixed;
+  z-index: 10;
+  top: var(--AppHeaderHeight);
+  bottom: 0;
+  left: 0;
+  overflow-y: auto;
+  padding-top: $kui-space-40;
+  padding-right: $kui-space-40;
+  border-right: $kui-border-width-10 solid $kui-color-border;
+  background-color:  $kui-color-background;
+}
+nav > ul > :is(li) {
```

Comment:
I changed this, but from what I understood its not a bug:

https://github.com/vuejs/core/discussions/8800#discussioncomment-6473724

> :is It is not an entity selector, but a modification selector, which is a supplement to the previous one, so there is no problem with the above compilation.
> Normal logic, because :is is not a entity selector, but a adjective selector. Same as :after

(I google translated the first line)